src/components/SearchResultItem/SearchResultItem.test.tsx

```diff
@@ -47,15 +66,15 @@ describe("SearchResultItem", () => {
     // let shortHeadingText = "hello";
     // let longHeadingText = `hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello `;
```

Comment:
Since you're here -- this can be deleted since text shortening will not happen in design system.  Could also be a small other ticket too.

Reply:
I can do this small change now, I was trying not to touch too much else but huge blocks of commented out code bothers me too